include/nbl/video/alloc/SubAllocatedDescriptorSet.h

```diff
@@ -217,17 +223,21 @@ class SubAllocatedDescriptorSet : public core::IReferenceCounted
 				// can we change it?
 				write.info = &descriptor;
 			}
-			infos.push_back(descriptor);
-			writes.push_back(write);
+			outDescriptorWrites[i] = write;
```

Comment:
why not call the time out version with a timepoint of `now()`

Reply:
ah ok there's an oberload of `multi_allocate` that does this, then the `try_multi_allocate` should be `protected`

Reply:
nevermind, I see the overload has a default time point for the wait. 

Discard everything I said, resolved.